content/api/api-elements.md

```diff
@@ -7,205 +7,1563 @@ author: Lawrence Lane
+     "field": "name", \
+     "order": "asc", \
+     "missing": "_last" \
+   }, \
+   "page": 0, \
+   "pageSize": 35, \
+   "startDate": "2020-01-24T01:00:00-05:00", \
+   "endDate": "2020-01-24T02:00:00-05:00", \
+   "elementTypes": { \
+     "and": false, \
+     "items": [ \
+       { \
+         "literal": true, \
+         "contains": true, \
+         "item": "EC2" \
+       } \
+     ] \
+   }, \
+   "sourceFilter": { \
+     "excludes": [ \
+       "metrics", \
+       "attributes" \
+     ] \
+   } \
+ }' 'https://app.metricly.com/elements/elasticsearch/elementQuery'
+```
+
+### Response Body
+
+The following response body found 2 elements with the **elementType** `EC2`.  You can easily find an element's **Id** and **FQN** by looking for `"type": "EC2"`.
+
+```
+{
+  "page": {
+    "content": [
+      {
+        "netuitiveTags": {
+          "n.analysis.status": null
+        },
+        "sourceTags": {
+          "Name": "test-element-abc1",
+          "n.collectors": "EC2"
+        },
+        "state": {
+          "netuitive.metrics.collected.percent": null
+        },
+        "eventCount": {
+          "total": 0,
+          "topCategory": null
+        },
+        "relations": [
+          {
+            "id": 11a031a5-011a-3976-84be-d0d1cbe281fe",
+            "dataSourceId": 31845,
+            "fqn": "501119301101:EBS:us-east-1:vol-01ab111bfa2c63111"
+          }
+        ],
+        "fqn": "11179301106:EC2:us-east-1:i-011adb1a4a45ab111",
+        "name": "Testing (10.13.10.83)",
+        "location": "us-east-1c",
+        "id": "91bb19b6-012c-1111-aede-dda4e12c111a",
+        "type": "EC2"
+      },
+      {
+        "netuitiveTags": {
+          "n.analysis.status": null,
+          "n.backfill.status": "false",
+          "n.state.maintenance": "false",
+          "n.state.maintenance_end": "-1"
+        },
+        "sourceTags": {
+          "Name": "Netuitive-colo: Domain Controller",
+          "app": "hq",
+          "n.collectors": "EC2"
+        },
+        "state": {
+          "netuitive.metrics.collected.percent": 100
+        },
+        "eventCount": {
+          "total": 0,
+          "topCategory": null
+        },
+        "relations": [
+          {
+            "id": "c6ccd4ac-ea3f-11be-ac98-11112aecf04c",
+            "dataSourceId": 31845,
+            "fqn": "502111101106:EBS:us-east-1:vol-9eba111e"
+          }
+        ],
+        "fqn": "112111101106:EC2:us-east-1:i-b4e87f28",
+        "lastProcessed": "2020-01-26T16:50:00Z",
+        "name": "Domain Controller (10.14.11.30)",
+        "location": "us-east-1a",
+        "id": "c48df6c2-111c-3542-11be-17b0472a7d12",
+        "type": "EC2"
+      },     
+    ],
+    "last": true,
+    "totalElements": 2,
+    "totalPages": 1,
+    "sort": null,
+    "first": true,
+    "numberOfElements": 2,
+    "size": 35,
+    "number": 0
+  }
+}
+```
+
+{{% /expand %}}
+
+---
+
+## POST to /elements/name/preview
```

Comment:
Could we add a small note somewhere indicating where a user can find the element ID?